test/openAPI/features/wallet_consent.feature

```diff
@@ -0,0 +1,33 @@
+@method=POST @endpoint=/linked-authorization/consent
+Feature: The endpoint to send the accepted user claims and permitted scopes.
+
+  @smoke @unit @positive
+  Scenario: Successfully sends the accepted user claims and permitted scopes smoke type test
+    Given Wants to send the accepted user claims and permitted scopes
+    And The linkedTransactionId is generated before POST /linked-authorization/consent
+    When Send POST /linked-authorization/consent request with given requestTime and linkedTransactionId
+    Then Receive a response from the /linked-authorization/consent endpoint
+    And The /linked-authorization/consent endpoint response should be returned in a timely manner 15000ms
+    And The /linked-authorization/consent endpoint response should have status 200
+
+  @positive
+  Scenario: Successfully sends the accepted user claims and permitted scopes
+    Given Wants to send the accepted user claims and permitted scopes
+    When Send POST /linked-authorization/consent request with given requestTime, linkedTransactionId, "openid" as permittedAuthorizeScopes, "given_name" as acceptedClaims
+    Then Receive a response from the /linked-authorization/consent endpoint
+    And The /linked-authorization/consent endpoint response should be returned in a timely manner 15000ms
+    And The /linked-authorization/consent endpoint response should have status 200
+    And The /linked-authorization/consent endpoint response should have content-type: application/json header
+    And The /linked-authorization/consent endpoint response should contain linkedTransactionId
+    And The /linked-authorization/consent endpoint response should match json schema without errors
+
+  @negative
+  Scenario: Not able to send the accepted user claims and permitted scopes because of invalid linkedTransactionId
+    Given Wants to check the correctness of the data
+    When Send POST /linked-authorization/consent request with given invalid linkedTransactionId
+    Then Receive a response from the /linked-authorization/consent endpoint
+    And The /linked-authorization/consent endpoint response should be returned in a timely manner 15000ms
+    And The /linked-authorization/consent endpoint response should have status 200
+    And The /linked-authorization/consent endpoint response should have content-type: application/json header
+    And The /linked-authorization/consent endpoint response should match json schema
+    And The /linked-authorization/consent response should contain errorCode property equals to "invalid_transaction_id"
```

Comment:
can we have 2 test scenarios for invalid_accepted_claim and  invalid_permitted_scope ?

Reply:
The issue is that in the api spec there is no information which accepted_claim and permitted_scope are valid, so I'm not sure how I can test it. Can you please help me find this information?

Reply:
I was hoping we had a enum on this, we can improve this in the spec and test in the future, marking this as resolved.